packages/checkout/sdk/src/smartCheckout/smartCheckout.ts

```diff
@@ -27,21 +24,35 @@ export const smartCheckout = async (
   provider: Web3Provider,
   itemRequirements: ItemRequirement[],
   transactionOrGasAmount?: FulfillmentTransaction | GasAmount,
+  includeFundingRoutesOnSufficient: boolean = false,
```

Comment:
What do you think about shortening this a little? maybe `allowRoutesOnSufficient` or `sufficientFundingRoutes`?

Reply:
Also just checking this would be a non-breaking change right? because we default to false here and we never returned the routes before if sufficient?